dist/manifest.json

```diff
@@ -7,9 +7,13 @@
   },
   "defaultLocale": "en",
   "private": false,
-  "location": "nav_bar",
+  "location": {
+    "support": {
+      "nav_bar": "assets/index.html"
+    }
+  },
   "version": "2.3.6",
```

Comment:
We need to update this version too. The question is should this be 2.4 or 3.0?

Reply:
@SofiaMargariti Given the tremendous changes of this branch I would go for 3.0

Reply:
@SofiaMargariti @nbasili I updated that with a new commit